package.json

```diff
@@ -30,70 +31,60 @@
   },
   "license": "MPL-2.0",
   "dependencies": {
-    "codemirror": "5.3.0",
+    "codemirror": "^5.65.0",
     "d3": "^3.1.5",
-    "es6-promise": "2.3.0",
-    "fetch": "1.0.0",
-    "flux": "~2.0.2",
-    "fontawesome": "^4.3.0",
     "jquery": "1.11.3",
     "jquery-ui": "1.12.1",
     "nunjucks": "^3.2.3",
-    "react": "0.13.3",
-    "underscore": "^1.13.1"
+    "underscore": "^1.13.2"
   },
   "devDependencies": {
-    "@babel/cli": "^7.12.16",
-    "@babel/core": "^7.12.16",
-    "@babel/preset-env": "^7.12.16",
-    "@babel/preset-react": "^7.12.13",
-    "@babel/register": "^7.12.13",
+    "@babel/core": "^7.16.7",
+    "@babel/plugin-transform-react-jsx": "^7.16.7",
+    "@babel/preset-env": "^7.16.7",
     "@mozilla-protocol/core": "10.0.1",
-    "autoprefixer": "^9.8.6",
+    "autoprefixer": "^10.4.1",
     "babel-loader": "^8.2.2",
-    "babel-plugin-module-resolver": "^4.1.0",
-    "babelify": "5.0.4",
-    "browser-sync": "^2.26.7",
-    "chai": "3.2.0",
+    "browser-sync": "^2.27.7",
+    "chai": "^4.3.4",
     "chai-lint": "0.1.1",
-    "color-string": "^1.6.0",
-    "concurrently": "^5.0.0",
-    "copy-webpack-plugin": "^9.0.0",
-    "css-loader": "^5.2.6",
-    "cssnano": "^4.1.11",
-    "eslint": "^8.1.0",
+    "color-string": "^1.9.0",
+    "concurrently": "^7.0.0",
+    "copy-webpack-plugin": "^10.2.0",
+    "css-loader": "^6.5.1",
+    "cssnano": "^5.0.14",
+    "eslint": "^8.6.0",
     "eslint-import-resolver-webpack": "^0.13.2",
-    "eslint-plugin-import": "^2.25.2",
-    "exports-loader": "^3.0.0",
-    "expose-loader": "^3.0.0",
+    "eslint-plugin-import": "^2.25.4",
+    "exports-loader": "^3.1.0",
     "glob": "^7.2.0",
-    "html-webpack-plugin": "^5.3.2",
-    "image-minimizer-webpack-plugin": "^2.2.0",
+    "html-webpack-plugin": "^5.5.0",
+    "image-minimizer-webpack-plugin": "^3.2.1",
+    "imagemin": "^8.0.1",
     "imagemin-optipng": "^8.0.0",
-    "imagemin-svgo": "^9.0.0",
-    "imports-loader": "^3.0.0",
+    "imagemin-svgo": "^10.0.1",
     "jsdom": "^19.0.0",
-    "kss": "^3.0.0-beta.25",
+    "kss": "^3.0.1",
     "locutus": "^2.0.15",
-    "mini-css-extract-plugin": "^1.6.0",
-    "mocha": "2.3.2",
-    "onchange": "^6.1.0",
+    "mini-css-extract-plugin": "^2.4.5",
+    "mocha": "^9.1.3",
+    "onchange": "^7.1.0",
     "path-parse": "^1.0.7",
-    "postcss": "^8.3.5",
-    "postcss-custom-properties": "^9.2.0",
-    "postcss-loader": "^6.1.1",
-    "sass": "^1.23.2",
-    "sass-loader": "^12.0.0",
+    "postcss": "^8.4.5",
+    "postcss-custom-properties": "^12.0.2",
+    "postcss-loader": "^6.2.1",
+    "react": "^0.13.3",
+    "sass": "^1.45.2",
+    "sass-loader": "^12.4.0",
     "sinon": "12.0.1",
-    "sinon-chai": "2.8.0",
+    "sinon-chai": "^3.7.0",
```

Comment:
node, sinon and chai upgrades: https://github.com/domenic/sinon-chai/releases/tag/v3.0.0